README.md

```diff
@@ -8,7 +8,7 @@
 
 AngularJS directive for working with Nokia Here Maps
 
-### 0.1.7 latest release
+### 0.1.8 latest release
```

Comment:
you don't need to changes this. 
I will do it only after the release

Reply:
ok, I'll return it back